src/main/java/training/endava/app/controller/PersonController.java

```diff
@@ -1,7 +1,61 @@
 package training.endava.app.controller;
 
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import training.endava.app.domain.Person;
+import training.endava.app.exceptions.PersonNotFoundException;
+import training.endava.app.filters.PersonDTO;
+import training.endava.app.mappers.PersonMapper;
+import training.endava.app.repository.PersonRepository;
+import training.endava.app.service.impl.PersonServiceImpl;
 
+import javax.validation.Valid;
+import java.util.List;
 
+@RestController
+@RequestMapping("/person")
 public class PersonController {
 
+    @Autowired
+    private PersonServiceImpl personService;
+
+    @GetMapping(value = "/id={ID}", produces = {MediaType.APPLICATION_XML_VALUE})
+    public ResponseEntity<?> getPersonById(@PathVariable(value="ID") Integer id){
+        Person person = this.personService.getPersonById(id);
+        if(person == null)
+            throw new PersonNotFoundException("Person you search for does not exists.");
+        else
+            return new ResponseEntity<PersonDTO>(PersonMapper.INSTANCE.personToPersonDTO(person),HttpStatus.OK);
```

Comment:
You can use ResponseEntity.ok(body). It does the same thing in the end,  but looks a bit more clean:)